src/utils/tree.js

```diff
@@ -13,29 +13,33 @@ var tf =
     },
 
     showTree: function(newickStr) {
-      var newick = this.require("biojs-io-newick");
+      var newick = require("biojs-io-newick");
```

Comment:
Have a look at what `loadTree` is for and what the `importFile` do:

https://github.com/wilzbach/msa/blob/master/src/utils/file.js#L87